colibre/registration.py

```diff
@@ -1690,6 +1690,42 @@ def register_stellar_birth_densities(self, catalogue):
     return
 
 
+def register_stellar_birth_temperatures(self, catalogue):
```

Comment:
Could we combine this function with register_stellar_birth_densities?

Reply:
Done